diff --git a/hug/api.py b/hug/api.py
index 7179c7a4..176e29c6 100644
--- a/hug/api.py
+++ b/hug/api.py
@@ -74,6 +74,7 @@ class InterfaceAPI(object):
 
     def __init__(self, api):
         self.api = api
+        self.api.future = True
 
 
 class HTTPInterfaceAPI(InterfaceAPI):
@@ -358,7 +359,7 @@ class HTTPInterfaceAPI(InterfaceAPI):
 
     def server(self, default_not_found=True, base_url=None):
         """Returns a WSGI compatible API server for the given Hug API module"""
-        falcon_api = self.falcon = falcon.API(middleware=self.middleware)
+        falcon_api = self.falcon = falcon.APP(middleware=self.middleware)
         if not self.api.future:
             falcon_api.req_options.keep_blank_qs_values = False
             falcon_api.req_options.auto_parse_qs_csv = True
diff --git a/setup.py b/setup.py
index 3d84b665..17e3da4e 100755
--- a/setup.py
+++ b/setup.py
@@ -96,7 +96,7 @@ setup(
     entry_points={"console_scripts": ["hug = hug:development_runner.hug.interface.cli"]},
     packages=["hug"],
     requires=["falcon", "requests"],
-    install_requires=["falcon==2.0.0", "requests"],
+    install_requires=["falcon", "requests"],
     setup_requires=["pytest-runner"],
     tests_require=["pytest", "mock", "marshmallow"],
     ext_modules=ext_modules,
